### DISCUSSION TITLE
请问我给Cargo.toml里tauri.features加上"devtools"为什么没有效果

### DISCUSSION THREAD

**Discussion by ACGNnsj** [tw93/Pake#174] [Q&A]
想启用开发者工具，找到Tauri官方文档上说这样设置，但是我自己照做之后还是不能打开
![image](https://user-images.githubusercontent.com/22112141/207829390-94e5f27f-ab73-431c-99c5-66d00fc1364d.png)

**Comment by ACGNnsj** [comment]
![image](https://user-images.githubusercontent.com/22112141/207833673-b2709821-a108-4142-a4b1-eacd53bc68da.png)
![image](https://user-images.githubusercontent.com/22112141/207833735-6df6bca3-63d2-40b1-aaa4-824b8f4eb376.png)
不知道是不是跟这个有关 不是很熟悉相关的配置

**Comment by tw93** [comment]
npm run dev:debug
